src/validation/__tests__/ArgumentsOfCorrectType-test.js

```diff
@@ -117,6 +117,70 @@ describe('Validate: Argument values of correct type', () => {
 
   });
 
+  describe('`null` values', () => {
```

Comment:
It's unnecessary to have a test for every scalar type - really you only need one case that expects a pass for the literal `null`, I'd title it `null into nullable type`. However you should also include a test for passing the literal `null` to a non-null type.